lib/linguist/languages.yml

```diff
@@ -4461,6 +4461,24 @@ SRecode Template:
   extensions:
   - ".srt"
   language_id: 335
+SSH Config:
+  type: data
+  group: INI
+  filenames:
+  - "ssh-config"
+  - "ssh_config"
+  - "sshconfig"
```

Comment:
I counted 409 such files on GitHub. [After that it looks like it's only `sshconfig.something`](https://github.com/search?p=41&q=filename%3Asshconfig&type=Code). I think 409 is enough given that there appear to be a 1 file -- 1 repository mapping.

Looks like [`sshconfig.vim` is quite popular](https://github.com/search?p=100&q=filename%3Asshconfig.vim&type=Code). Maybe we should add it as well?

Reply:
sshconfig.vim looks like a vim file rather than sshconfig and it's already highlighted correctly.

Reply:
There seems to be [252 results with sshconfig.snip](https://github.com/search?p=5&q=filename%3Asshconfig.snip&type=Code) thanks to neosnippets. Other extensions seem to in their respective syntaxes if it's supported.

Reply:
Ah, right! I missed that.

Let's add `sshconfig.snip` then.

Reply:
I removed the extensions and added sshconfig.snip to filenames but now getting couple of conflicts that I don't know how to resolve as they seem to be coming from places where I haven't touched.